utils.py

```diff
@@ -129,17 +129,17 @@ def get_exploration_components_from_zip(zip_file_contents):
     """Gets the (yaml, assets) from the contents of an exploration zip file.
 
     Args:
-      zip_file_contents: a string of raw bytes representing the contents of
-        a zip file that comprises the exploration.
+        zip_file_contents: a string of raw bytes representing the contents of
+            a zip file that comprises the exploration.
 
     Returns:
-      a 2-tuple, the first element of which is a yaml string, and the second
-      element of which is a list of (filepath, content) 2-tuples. The filepath
-      does not include the assets/ prefix.
+        a 2-tuple, the first element of which is a yaml string, and the second
+        element of which is a list of (filepath, content) 2-tuples. The filepath
+        does not include the assets/ prefix.
 
     Raises:
-      Exception: if the following condition doesn't hold: "There is exactly one
-        file not in assets/, and this file has a .yaml suffix".
+        Exception: if the following condition doesn't hold: "There is exactly
+        one file not in assets/, and this file has a .yaml suffix".
```

Comment:
Indent last line by 4 spaces as it is the continuation line.

Reply:
Done :+1: